content/en/user-guide/aws/lambda/index.md

```diff
@@ -267,7 +267,7 @@ In the old Lambda provider, Lambda functions were executed within the LocalStack
 
 If you encounter the following error message, you may be using the local executor mode:
 
-{{< tabpane >}}
+{{< tabpane lang="bash" >}}
```

Comment:
Did the lang move to the tabpane from tab or do we need it in both places?
It's a bit weird in such scenarios where we have two inconsistent ones 🤔

Reply:
The one on the tabpane is the default, the tabs overrides it. We however need to set a default, or the persistence key (to persist tab selection) defaults to the lang of the tabs, which might be identical.